src/numpy.jl

```diff
@@ -64,7 +64,7 @@ function npyinitialize()
     numpy = pyimport("numpy")
 
     # emit a warning if both Julia and NumPy are linked with MKL (#433)
-    if LinearAlgebra.BLAS.vendor() === :mkl &&
+    if :mkl in LinearAlgebra.BLAS.get_config().build_flags &&
```

Comment:
Doesn't work with Julia 1.0, which we still declare compatibility with.

Reply:
I have committed a revision to my colbec-patch-1 branch, trying to get it reflected in the PR